cylc/flow/scripts/cylc.py

```diff
@@ -663,3 +671,30 @@ def main():
                 if opts.version:
                     cmd_args.append("--version")
                 execute_cmd(command, *cmd_args)
+
+
+def handle_missing_dependency(
```

Comment:
You could probably unit test this bit fairly trivially, which would at least leave the untested surface smaller.

Reply:
Can patch `iter_entry_points` to return something with a `resolve` method that raises `ModuleNotFoundError`.